packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -396,6 +401,22 @@ export default class TabAdvanced extends Component {
                       <Toggle value={pool.auto_poweron} onChange={this._onChangeAutoPoweron} />
                     </td>
                   </tr>
+                  <tr>
+                    <th>{_('migrationCompression')}</th>
+                    <td>
+                      <Tooltip
+                        content={
+                          pool.migrationCompression === undefined ? _('migrationCompressionDisabled') : undefined
```

Comment:
To avoid duplication, you can create for `pool.migrationCompression === undefined`.
E.g. `const isMigrationCompressionAvailable = pool.migrationCompression === undefined`